site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm

```diff
@@ -272,8 +272,15 @@
 					#if ($!published)
 						#if ($additionalAccess.size() > 0)
 							<ul class="attachList" style="margin:0">
-								#if ($!siteJoinable)
-									<li>$tlang.getString("sitegen.siteinfolist.sitecan")</li>
+								#if ($!siteJoinable && $joinLimitedAccountTypes.size() > 0)
+									<li>$tlang.getFormattedMessage("sitegen.siteinfolist.sitecan", $uiService)</li>
+									#foreach( $accountType in $joinLimitedAccountTypes )
```

Comment:
Should this contain a nested `<ul>` as in this situation we have one list inside another don't we?